### PR TITLE
8551 problems with svg images as image field

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: dotCMS/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp [hunk 1]
```diff
@@ -715,7 +715,15 @@ dojo.require("dojox.layout.ContentPane");
 				}
 
 				var thumbNailImg = document.createElement("img");
-				thumbNailImg.setAttribute("src","/contentAsset/image/"+data['contentletInode']+"/fileAsset/?byInode=1&filter=Thumbnail&thumbnail_w=300&thumbnail_h=300");
+				var thumbnailImage;
+
+				if(fieldRelatedData.fileName.indexOf('svg') <0) {
```
Comment (jgambarios) [inline dotCMS/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp]: Just to be sure maybe change it to **.endsWith(".svg")** ?

File: dotCMS/html/portlet/ext/contentlet/contentlet_versions_inc.jsp [hunk 2]
```diff
@@ -115,8 +115,12 @@
 			 	<% if (!working && canEdit && !live) { %>
 			 		<a  href="javascript: selectVersion('<%= vinode %>');">
 			 	<%} %>
-			 	<img src="/contentAsset/image/<%=vinode %>/fileAsset/?byInode=1&filter=Thumbnail&thumbnail_h=125&thumbnail_w=125" style="width:150px;height:150px;border:1px solid silver;padding:3px;"></a>
 
+				<%if(ident.getAssetName().toLowerCase().equals("svg")){%>
```
Comment (jgambarios) [inline dotCMS/html/portlet/ext/contentlet/contentlet_versions_inc.jsp]: Can we trust in the asset name in order to identify something as a **svg** image?

File: dotCMS/html/js/dotcms/dijit/image/ImageEditor.js [hunk 3]
```diff
@@ -136,7 +136,13 @@ dojo.declare("dotcms.dijit.image.ImageEditor", dijit._Widget,{
             if(thumbUrl.indexOf("?") <0){
                 //thumbUrl+="?";
             }
-            thumbUrl+= "/filter/Thumbnail/thumbnail_w/" + this.thumbnailWidth+ "/thumbnail_h/"+ this.thumbnailHeight;
+
+            if(this.saveAsFileName.indexOf('svg') <0) {
```
Comment (jgambarios) [inline dotCMS/html/js/dotcms/dijit/image/ImageEditor.js]: Just to be sure maybe change it to **.endsWith(".svg")** ?

**Comment by freddyucv** [comment]
@jgambarios Listo arreglado todos los comentarios

**Comment by jgambarios** [comment]
Waiting for some toLowerCase before some string comparisons in order to avoid future problems.

**Pull request merged**